imports/ui/components/GeoSuggest.jsx

```diff
@@ -33,18 +33,23 @@ export class GeoSuggest extends React.Component {
   };
 
   render() {
+    /* -----------------------------------------------------------
+   These options will bias the autocomplete predictions toward 
+         Vancouver, Canada with a radius of 200 meters.
+	-------------------------------------------------------------*/
     const searchOptions = {
       location: new google.maps.LatLng(49.290338, -123.134112),
-      radius: 100
+      radius: 200
```

Comment:
![image](https://user-images.githubusercontent.com/25490855/60524267-1ce5ca00-9ca1-11e9-8dac-0292e3aab0a5.png)

Can we have it be all of the lower mainland?